byol-license/func.ts

```diff
@@ -7,7 +7,7 @@ import {
     AzureFunctionResponse,
     AzurePlatformAdaptee,
     AzurePlatformAdapter
-} from '@fortinet/fortigate-autoscale/azure';
+} from '@fortinet/autoscale/fortigate-autoscale/azure';
```

Comment:
this is incorrect, should be the previous

Reply:
fixed